### DISCUSSION TITLE
can't install zygisknext, any help

### DISCUSSION THREAD

**Discussion by annex88** [bmax121/APatch#367] [Q&A]
install zygisknext 1.0.2-1.0.4 failed from apatch manager,any mistakes I make？
 the log:
 Preparing image
- Module size: 3.25 MB
e2fsck 1.45.4 (23-Sep-2019)
Error: No such file or directory (os error 2)
- Error: No such file or directory (os error 2)

**Comment by pomelohan** [comment]
Open issues instead
